pitch.md

```diff
@@ -14,31 +18,82 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Often when writing robot code, quite a bit of work needs to go into writing the
+boiler plate for a state machine or interrupts or whatever structure is needed
+to get the actual logic implemented. It would be really nice for both
+inexperienced robot developers and people trying to write robot code quickly to
+be able to just specify the logic, and let someone else take care of the boiler
+plate.
+
+I've observed freshmen in E11 (autonomous vehicles) trying to write C++ for
+their robots. They neither know C++ very well nor have a great idea about how to
+write good robot code. A DSL can help solve both of these issues. The syntax
+could be somewhat nicer than C++'s (not a hard task), and the language can help
+steer programmers into using good design constructs by making them easy. In the
```

Comment:
I really like the idea of providing students a sort of educational language that teaches them "good design constructs" and makes 'robot coding' less miserable, which I think will lead to more enjoyment of the course and a decreased tendency toward bad coding habits.